keyboards/keebio/levinson/keymaps/issmirnov/keymap.c

```diff
@@ -0,0 +1,159 @@
+#include QMK_KEYBOARD_H
+
+#include "issmirnov.h"
+#include "quantum_keycodes.h"
+
+#include "tap_tog.h"
+
+#ifdef AUDIO_ENABLE
+  #include "audio.h"
+  #include "sounds.h"
+#endif
+
+
+#ifdef RGBLIGHT_ENABLE
+  #include "rgb.h"
+#endif
+
```

Comment:
Noted, thank you. Part of my workflow is setting `AUDO_ENABLE = false` when doing frequent flashing to avoid annoying coworkers. Is there a better way to do this? Do the audio files get included regardless, or does the `AUDIO_ENABLE = no` do some additional magic during the automatic QMK inclusions?

Reply:
Actually, could you clarify your comment please? When I remove the block entirely, I get compilation errors due to missing definitions for `keyboard_post_init_rgb` and `song_goodbye`, etc. Can you help me understand how this can be removed?

Reply:
If these are includes from your keymap, then yeah, they should stay. 

If they're from your userspace, then ideally, you should move them to your userspace's header file, so you only need to add an include for a single file, rather than a half dozen

Reply:
Ah. At the moment, they are just for this keymap. The levinson is the only one with RGB an Audio. If I get a second one, I will definitely move it to the userspace (like I did with the key layouts).

Thanks!